### PR TITLE
fix tests on windows

### PR DESCRIPTION
**Pull request by devoncarew**
Related to https://github.com/flutter/tools/issues/36, fix two tests - our tests now pass on windows.

@abarth, fyi @TheBosZ

### CODE REVIEW

File: lib/src/process.dart [hunk 1]
```diff
@@ -64,6 +64,12 @@ String runCheckedSync(List<String> cmd) =>
 /// Run cmd and return stdout.
 String runSync(List<String> cmd) => _runWithLoggingSync(cmd);
 
+/// Return the platform specific name for the given Dart SDK binary. So, `pub`
+/// ==> `pub.bat`.
+String sdkBinaryName(String name) {
+  return Platform.isWindows ? '${name}.bat' : name;
+}
```
Comment (abarth) [inline lib/src/process.dart]: Does this mean we can remove the `runInShell` patch that we added recently?

**Comment by abarth** [comment]
LGTM

File: lib/src/process.dart (hunk 1 above)
Comment (devoncarew) [inline lib/src/process.dart]: I'll confirm that and remove if its not needed -

**Comment by TheBosZ** [comment]
Fantastic! Starting to really come together. I'll be sure to do more tests
in the future.

**Comment by devoncarew** [comment]
Thanks for your contributions @TheBosZ. This cleans up our tests, but with your changes I was able to launch a flutter app from windows. Pretty cool!

**Pull request merged**